src/MainWindow.vala

```diff
@@ -70,11 +70,19 @@ public class Camera.MainWindow : Hdy.ApplicationWindow {
 
         add (window_handle);
 
+        camera_view.camera_added.connect (header_bar.add_camera_option);
+        camera_view.camera_removed.connect (header_bar.remove_camera_option);
+        header_bar.request_camera_change.connect (camera_view.change_camera);
+
         timer_running = false;
 
         camera_view.start ();
 
         show_all ();
+
+        //  for (int i = 0; i < camera_view.get_cameras (); i++) {
+        //      header_bar.add_camera_options (camera_view.get_camera (i));
+        //  }
```

Comment:
Looks like some commented code got left in here. :)

Reply:
Done. Sorry about that.